{
	"id": "CVE-2023-37453",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2023-37453",
				"description": "An issue was discovered in the USB subsystem in the Linux kernel through 6.4.2. There is an out-of-bounds and crash in read_descriptors in drivers/usb/core/sysfs.c.",
				"references": [
					{
						"source": "CVE",
						"url": "https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=1e4c574225cc5a0553115e5eb5787d1474db5b0f"
					},
					{
						"source": "CVE",
						"url": "https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=85d07c55621676d47d873d2749b88f783cd4d5a1"
					},
					{
						"source": "CVE",
						"url": "https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=de28e469da75359a2bb8cd8778b78aa64b1be1f4"
					},
					{
						"source": "CVE",
						"url": "https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=ff33299ec8bb80cdcc073ad9c506bd79bb2ed20b"
					},
					{
						"source": "CVE",
						"url": "https://lore.kernel.org/all/000000000000c0ffe505fe86c9ca%40google.com/T/"
					},
					{
						"source": "CVE",
						"url": "https://lore.kernel.org/all/000000000000e56434059580f86e%40google.com/T/"
					},
					{
						"source": "CVE",
						"url": "https://syzkaller.appspot.com/bug?extid=18996170f8096c6174d0"
					},
					{
						"source": "mitre",
						"url": "https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=1e4c574225cc5a0553115e5eb5787d1474db5b0f"
					},
					{
						"source": "mitre",
						"url": "https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=85d07c55621676d47d873d2749b88f783cd4d5a1"
					},
					{
						"source": "mitre",
						"url": "https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=de28e469da75359a2bb8cd8778b78aa64b1be1f4"
					},
					{
						"source": "mitre",
						"url": "https://git.kernel.org/cgit/linux/kernel/git/torvalds/linux.git/commit/?id=ff33299ec8bb80cdcc073ad9c506bd79bb2ed20b"
					},
					{
						"source": "mitre",
						"url": "https://lore.kernel.org/all/000000000000c0ffe505fe86c9ca%40google.com/T/"
					},
					{
						"source": "mitre",
						"url": "https://lore.kernel.org/all/000000000000e56434059580f86e%40google.com/T/"
					},
					{
						"source": "mitre",
						"url": "https://syzkaller.appspot.com/bug?extid=18996170f8096c6174d0"
					}
				],
				"modified": "2024-08-02T17:16:29.576Z"
			}
		}
	],
	"data_source": {
		"id": "mitre-v5",
		"raws": [
			"vuls-data-raw-mitre-v5/2023/CVE-2023-37453.json"
		]
	}
}
